### PR TITLE
Update CircleCI config to test ansible playbook

### PR DESCRIPTION
**Pull request by guiadco**
### Current behaviour

As a developper i want to see the result of my modification on a test machine different from vagrant box

---
### Expected behaviour

See the completion of the playbook on CircleCI

---
### Modifications

> What are the changes involved to match with the expected behaviour ?

-  [x] Update CircleCI Conf with a Virtual Machine and the Playbook to test it

---
### Status

- [x] Implementation
- [x] Test coverage

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,11 +50,32 @@ jobs:
...
+  test-install:
+    machine: true
+    working_directory: ~/repo
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "3e:42:39:0c:71:79:a2:20:13:68:de:3a:a7:0d:a9:b5"
+      - run: sudo apt-get -y update
+      - run: sudo apt-add-repository ppa:ansible/ansible
+      - run: git clone https://github.com/actiniumio/allspark.git .
```
Comment (Xide) [inline .circleci/config.yml]: You can use the circleCI `checkout` step. https://circleci.com/docs/2.0/configuration-reference/#checkout

File: .circleci/config.yml [hunk 2]
```diff
@@ -50,11 +50,32 @@ jobs:
...
+      - run: sudo apt-get -y update
+      - run: git checkout $CIRCLE_BRANCH
+      - run: sudo apt-get install -y ansible
+      - run: sudo ansible --version || true
+      - run: sudo ansible-playbook -i hosts install.yml -vvv
```
Comment (Xide) [inline .circleci/config.yml]: Why do u use sudo with ansible ?

**Comment by Xide** [comment]
Hi,
Could you name your steps with the syntax
```yaml
- run:
  name: Your step name
  command: <run command>
```.

It would make the output of the workflow : https://circleci.com/gh/actiniumio/allspark/462?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: .circleci/config.yml (hunk 1 above)
Comment (guiadco) [reply .circleci/config.yml]: actually i use git over HTTPS:
`If you require doing git over HTTPS you should not use this step as it configures git to checkout over ssh.`
I wil see to change to git over ssh

**Pull request merged**